frontend/app/components/debug-payload.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+import { faCheck, faClipboard } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { Button } from './buttons';
+
+export type debugPayloadtProps = {
```

Comment:
I know this has already been merged.. but there's a typo here.

Reply:
Good catch, can be fixed.